web/src/scheduled_messages_ui.js

```diff
@@ -59,6 +59,7 @@ export function open_scheduled_message_in_compose(scheduled_msg, should_narrow_t
             type: scheduled_msg.type,
             private_message_recipient: recipient_emails.join(","),
             content: scheduled_msg.content,
+            keep_composebox_empty: true,
```

Comment:
This option seems like it's not required; the `content` option is doing the job for us, right?

Reply:
`content` should already handle this case yes, but I was imagining it might be clearer to also pass `keep_composebox_empty` more explicitly? And also maybe would help anyone doing similar things in the future remember about making sure the composebox is kept empty. 

I don't feel super strongly about having or not having it there. Let me know if you prefer I remove it.

Reply:
Yeah, it's fine for it to be there, just wanted to make sure I understood how this works correctly.